src/sentry/migrations/0425_add_org_member_id_to_organizationmembermapping_table.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.28 on 2023-04-21 16:43
+
+from django.db import migrations
+
+import sentry.db.models.fields.bounded
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0424_drop_group_forecast_table"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="organizationmembermapping",
+            name="organizationmember_id",
+            field=sentry.db.models.fields.bounded.BoundedBigIntegerField(db_index=True, null=True),
```

Comment:
Having this be a `HybridCloudForeignKey` could be good as we could use tombstones to propagate the deletes from regions to control. That would save us having to build RPC methods.

Reply:
We'll be unable to use `HybridCloudForeignKey` here as `organizationmember_id` are only unique to their own region.